test/OAuth2/Encryption/JwtTest.php

```diff
@@ -45,28 +45,26 @@ public function testJwtUtil($client_id, $client_key)
 
         $encoded = $jwtUtil->encode($params, $this->privateKey, 'RS256');
 
-        // test BC behaviour of trusting the algorithm in the header
-        $payload = $jwtUtil->decode($encoded, $client_key);
-        $this->assertEquals($params, $payload);
-
-        // test BC behaviour of not verifying by passing false
-        $payload = $jwtUtil->decode($encoded, $client_key, false);
-        $this->assertEquals($params, $payload);
-
         // test the new restricted algorithms header
         $payload = $jwtUtil->decode($encoded, $client_key, array('RS256'));
         $this->assertEquals($params, $payload);
     }
 
+    /**
+     * @expectedException \UnexpectedValueException
+     */
     public function testInvalidJwt()
     {
         $jwtUtil = new Jwt();
 
-        $this->assertFalse($jwtUtil->decode('goob'));
-        $this->assertFalse($jwtUtil->decode('go.o.b'));
+        $jwtUtil->decode('goob');
+        $jwtUtil->decode('go.o.b');
```

Comment:
this second line requires another test, as the first line will throw an exception and therefore halt execution